test/unit/offline-support/offline_support_api.test.ts

```diff
@@ -2325,6 +2325,42 @@ describe('OfflineDBSyncManager', () => {
         expect(upsertUserSyncStatusSpy).toHaveBeenCalled();
       });
 
+      it('do not resets DB if sync API throws an AxiosError with request timeout', async () => {
+        const recentDate = new Date();
+        recentDate.setDate(recentDate.getDate() - 10);
+        getLastSyncedAtSpy.mockResolvedValueOnce(recentDate.toString());
+
+        const axiosError = {
+          isAxiosError: true,
+          code: 'ECONNABORTED',
+          response: { data: { code: 4 } },
+        } as AxiosError<APIErrorResponse>;
+
+        syncApiSpy.mockRejectedValueOnce(axiosError);
+
+        await (syncManager as any).sync();
+
+        expect(resetDBSpy).not.toHaveBeenCalled();
+        expect(upsertUserSyncStatusSpy).not.toHaveBeenCalled();
+      });
+
+      it('do not reset DB if sync API throws a BE error with response timeout', async () => {
+        const recentDate = new Date();
+        recentDate.setDate(recentDate.getDate() - 10);
+        getLastSyncedAtSpy.mockResolvedValueOnce(recentDate.toString());
+
+        const axiosError = {
+          response: { data: { code: 23 } },
+        } as AxiosError<APIErrorResponse>;
+
+        syncApiSpy.mockRejectedValueOnce(axiosError);
+
+        await (syncManager as any).sync();
+
+        expect(resetDBSpy).not.toHaveBeenCalled();
+        expect(upsertUserSyncStatusSpy).not.toHaveBeenCalled();
+      });
+
```

Comment:
In the test just below this one, please add an additional check for the `channelManager.state.isLoading` value (it should now be `false` with the recent fixes)